spec/spec_helper.rb

```diff
@@ -13,6 +14,10 @@
   config.include Helpers::Requests
   config.color = true
   config.tty = true
+
+  config.before(:each) do
+    stub_do_not_mail_list_exists(body: read_fixture('do_not_mail_list', 'do_not_mail_not_found.json'))
```

Comment:
do you need to stub ever test with this, or could it be added just to some of the tests?

Reply:
I could. I'll see what we can do.